### ISSUE TITLE
improve hierarchy and link structure in header on pages other than home: 3

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#326]
Currently, all our pages use a common header in which we find:

```html
<header id="portal-top" role="banner">
    <h1 id="site-title">
        <a href="http://isaw.nyu.edu" title="Institute for the Study of the Ancient World" accesskey="1">Institute for the Study of the Ancient World</a></h1>
    <div class="identity"><a href="/">ISAW</a></div>
    <h3 class="trigger">Menu</h3>
    <div id="main-navigation">
⋮
```

Changes to structure and tagging in this portion of the header are required in order to make our header more accessible. **Note that visual look and feel of the header at all responsive page widths must not change as a result of this work**; therefore, modifications to CSS rules **may** be required to preserve current look and feel once elements are changed. 

## The following changes need to be made on the home page (/):

```html
<header id="portal-top" role="banner">
    <a href="/" accesskey="1">
        <div class="identity">ISAW</div>
        <h1 id="site-title">Institute for the Study of the Ancient World</h1>
    </a>
    <div class="trigger">Menu</div>
    <div id="main-navigation">
⋮
```

## The following changes need to be made on all other pages:

```html
<header id="portal-top" role="banner">
    <a href="/" accesskey="1">
        <div class="identity">ISAW</div>
        <div id="site-title">Institute for the Study of the Ancient World</div>
    </a>
    <div class="trigger">Menu</div>
    <div id="main-navigation">
⋮
```

## These changes achieve the following accessibility goals:

 - Eliminate redundant adjacent links in navigation.
 - Stop using ```<h1>``` for something that is not the title/heading of the current context
 - Stop using ```<h3>``` for something that is not part of the content hierarchy on the page

## Important note regarding ```<h3 class="trigger">```

This element is hidden via CSS in the desktop view, but exposed in tablet and mobile responses. There is an entire epic (#265) whose purpose is making the mobile menu more accessible. If this ticket is undertaken after the completion of epic #265, care must be taken not to break that solution. If the mobile menu solution will not work with ```<div class="trigger">``` then the appropriate solution to #265 should be kept, provided it does not involve the use of heading elements.

## Related epics:

#230 normalize page heading hierarchies
#324 accessibility for the "People Listing"

**Comment by cguardia** [comment]
This is on staging now.

**Comment by paregorios** [comment]
It looks like two out of the three things we need are done:

 - [x] Get rid of redundant ```<a>``` links to site root in header.
 - [x] Don't use ```<1>``` for the site name on pages other than home; use it instead for the page title.
 - [ ] Stop using ```<h3>``` for ```class="trigger"``` in the header.

@cguardia was there a reason we couldn't move from ```<h3 class="trigger">``` to something like ```<div class="trigger">``` as suggested above?

**Comment by cguardia** [comment]
No, I just forgot to do it. Will make the change.

**Comment by cguardia** [comment]
On staging now.

**Comment by paregorios** [comment]
That's got it! Please merge to master and deploy to production.

**Comment by paregorios** [comment]
@skleinfeldt It looks to me like this has not yet been deployed to production. Is that correct?

**Comment by skleinfeldt** [comment]
@paregorios I have no idea but would be inclined to trust your observations.

@alecpm can you deploy this along with your fixes to #242 and #271 when you make them? Unless @cguardia has already deployed?

**Comment by paregorios** [comment]
@alecpm I am not seeing the header changes on production. I have dropped my browser cache and I have also restarted varnish on the server. Please advise.

**Comment by alecpm** [comment]
@cguardia This was merged to master in its entirety? Does it require any upgrade steps? I also restarted varnish and reloaded the CSS registry.

**Comment by alecpm** [comment]
Should be fixed now. I needed to re-save the theme settings to force it to use the new diazo rules.xml. Normally that changed would have shown up with a restart (or when running in debug mode), but because there were no python code changes in this batch of fixes I hadn't restarted the instances.

**Comment by paregorios** [comment]
That's got it!
